packages/webhooks/lib/types.ts

```diff
@@ -0,0 +1,36 @@
+import type { SyncType, FailedConnectionError, AuthOperationType, AuthModeType, SyncResult } from '@nangohq/types';
+import type { WebhookType } from './enums.js';
+
```

Comment:
that would be great if types does not export any code to avoid potential mistakes, if you both want to I'm not going to against it but if we can avoid it I'm happy :D

Reply:
I agree that `types` should not export code and I spent a lot of time with this PR changing enums to interfaces to avoid just that.

Reply:
yeah and imo I doesn't make the codebase any less good (or better tbf) it's just one less thing to import